src/Aspire.Cli/Program.cs

```diff
@@ -620,8 +633,8 @@ public static async Task<int> Main(string[] args)
     {
         System.Console.OutputEncoding = Encoding.UTF8;
         var rootCommand = GetRootCommand();
-        var result = rootCommand.Parse(args);
-        var exitCode = await result.InvokeAsync().ConfigureAwait(false);
-        return exitCode;
+        var config = new CommandLineConfiguration(rootCommand);
+        config.EnableDefaultExceptionHandler = true;
+        return await config.InvokeAsync(args).ConfigureAwait(false);
```

Comment:
FYI @mitchdenny - since we are in an application here (and not a library), you don't have to `ConfigureAwait(false)`.

I can update 

https://github.com/dotnet/aspire/blob/886f29357510ed46ea118afa91f128c9f783cda1/.editorconfig#L476-L478

to add the Aspire.Cli.